akka-sample-sharding-scala/killrweather-fog/src/main/scala/sample/killrweather/fog/WeatherStation.scala

```diff
@@ -53,7 +53,7 @@ private class WeatherStation(context: ActorContext[WeatherStation.Command], wsid
   }
   private val stationUrl = s"http://${settings.host}:${httpPort}/weather/$wsid"
 
-  def running(httpPort: Int): Behavior[WeatherStation.Command] = {
+  val running: Behavior[WeatherStation.Command] = {
```

Comment:
```suggestion
  def running: Behavior[WeatherStation.Command] = {
```

Let's keep it a def (even if a val in practice works fine) as that makes the design pattern with a wrapping class to capture dependencies and then a method to create the initial behavior a bit more clear.